chia/full_node/coin_store.py

```diff
@@ -574,7 +583,8 @@ async def rollback_to_block(self, block_index: int) -> dict[bytes32, CoinRecord]
             )
             for row in rows:
                 coin = self.row_to_coin(row)
-                record = CoinRecord(coin, row[0], uint32(0), row[2], row[6])
+                coinbase = False if row[2] == 0 else True
```

Comment:
I would suggest this or the alternative below.
```suggestion
                coinbase = row[2] != 0
```